pkg/controllers/openshift-route-controller-manager.go

```diff
@@ -114,37 +114,37 @@ func (s *OCPRouteControllerManager) Run(ctx context.Context, ready chan<- struct
 		klog.Fatalf("failed to apply openshift namespaces %v", err)
 	}
 	if err := assets.ApplyClusterRoles([]string{
-		"core/ingress-to-route-controller-clusterrole.yaml",
-		"core/route-controller-informer-clusterrole.yaml",
-		"core/route-controller-tokenreview-clusterrole.yaml",
+		"rbac/ingress-to-route-controller-clusterrole.yaml",
```

Comment:
since we do not always have a prefix `route-controller`, I would be in favor of having even more granular approach like `rbac/route-controller-manager/` to distinguish the files. But we are not doing this in our operators and is probably okay in a current state with not as many RBACs